tests/testthat/test-run.R

```diff
@@ -741,3 +735,19 @@ test_that("Can use connections with two databases", {
   p <- orderly_commit(id, root = path)
   expect_true(file.exists(file.path(p, "mygraph.png")))
 })
+
+
+test_that("prevent duplicate filenames", {
```

Comment:
This was never done before, but I'm doing it opportunistically.  Enforcing this will help with eventually deduplicating the archive, and reasoning about where orderly's files come from